desktop/src/jvmMain/kotlin/HarmonyColorPickerScreen.kt

```diff
@@ -65,7 +67,7 @@ fun HarmonyColorPickerScreen() {
             harmonyMode = harmonyMode.value,
             modifier = Modifier.defaultMinSize(minHeight = 300.dp, minWidth = 300.dp),
             onColorChanged = { hsvColor ->
```

Comment:
This doesn't seem to be compiling for desktop, is it working for you on desktop?

Reply:
I see the issue and have pushed it @riggaroo.  Sorry I didn't notice the desktop runner at all 🤦‍♂️